src/components/sidebar/linkView/LinkView.tsx

```diff
@@ -68,26 +117,32 @@ class LinkView extends React.Component<ILinkViewProps, ILinkViewState> {
                     <div className={s.flexInnerRow}>
                         <InputContainer
                             label='REITTITUNNUS'
-                            placeholder='1016'
+                            value={this.state.routePathLink!.routeId}
                         />
                         <InputContainer
                             label='SUUNTA'
-                            placeholder='Suunta 1'
+                            value={`Suunta ${routePath ? routePath.direction : '?'}`}
```

Comment:
I'm having a hard time understading why this is a text input field and not a dropdown or number field (in the db there are only 1, 2 and 3 values for suusuunta)

Reply:
True, but this will change, so I sat the input to disabled